apps/api/src/main.mts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import cors from 'cors';
+import express, { Request, Response } from 'express';
+
+export const port = process.env.PORT;
+
+// TODO: Fix Types package so we can import from main instead of specific file
+import { demoApplication } from 'pcgl-daco/packages/data-model/src/main.mts';
+
+const app = express();
+
+app.use(cors());
+
+app.get('/', (_req: Request, res: Response) => {
+	res.send('Hello World!');
+});
+
+app.get('/applications', cors(), (_req: Request, res: Response) => {
```

Comment:
setting `cors` middleware at app level means it's enabled for all app handlers
https://expressjs.com/en/resources/middleware/cors.html
I think the first and second example both made it in

Reply:
Ooh sorry I see, yeah it looks like I mixed their examples. Updated.